cfpb/consumerfinance.gov
# Ask: Add feature flag for date label change

Wraps Ask date label change in feature flag.

## Additions

- Feature flag for Ask date label change.


## How to test this PR

1. Check out branch
2. Navigate to an [Ask answer page](http://localhost:8000/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/) and verify that its date label is `Updated` for the English page and `Actualizado` for the Spanish page
3. In settings, turn the feature flag `ASK_UPDATED_DATE_LABEL` on
4. Navigate to an [Ask answer page](http://localhost:8000/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/) and verify that its date label is now `Last reviewed` for the English page and `Revisado` for the Spanish page
